src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,29 @@
+.catalog {
+  --column-count: 1;
+
+  display: grid;
+  box-sizing: border-box;
+  padding: 50px 40px;
+  grid-template-columns: repeat(auto-fill, 200px);
+  grid-template-rows: repeat(2, 1fr);
```

Comment:
The `grid-template-rows` property is set to `repeat(2, 1fr)`. This means that two rows will always be created, regardless of the number of items. If you intended to have a flexible number of rows depending on the number of items, consider using `auto-fill` or `auto-fit` instead.